src/app/(library)/layout.tsx

```diff
@@ -1,9 +1,22 @@
+import { SignUpBanner } from "@/auth/features/sign-up";
+import { NavBar } from "@/library/widgets/navBar";
+import { Player } from "@/player/widgets/player";
+import { getServerSession } from "@/shared/auth/next";
 import { MainLayout } from "@/shared/ui/layouts";
 
-export default function Layout({
+export default async function Layout({
   children,
 }: Readonly<{
   children: React.ReactNode;
 }>) {
-  return <MainLayout>{children}</MainLayout>;
+  const session = await getServerSession();
+
+  return (
+    <MainLayout
+      footer={session ? <Player /> : <SignUpBanner />}
+      navBar={<NavBar />}
+    >
+      {children}
+    </MainLayout>
+  );
```

Comment:
Just a note about UX / performances: It is not really ideal to block the entire layout (`await getServerSession()`). 

We should handle this verification deeper in the tree. We can (and will) improve it later